### PR TITLE
updated getAllDomains function description with information about exp…

### PR DESCRIPTION
**Pull request by akashrdev**
updated getAllDomains function description with information about expected response

### CODE REVIEW

File: sns-book/src/domain-name/domain-find-for-owner.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+
+```js
+import { getAllDomains } from "@bonfida/spl-name-service";
+
+// ...
+
+const domains = await getAllDomains(connection, user);
+```
+
+The function above will return an array of 2 Public Keys. Use the toBase58() method to convert the keys into a human readable format.
```
Comment (dr497) [inline sns-book/src/domain-name/domain-find-for-owner.md]: 1. Why did you write `2 Public Keys`? I think it's just an array of public keys
2. `Use the toBase58() method to convert the keys into a human readable format` human readable reads a bit weird to me in this context as it usually refers to reverses. `Use the toBase58() method to convert public keys to a base 58 encoded strings` seems a bit more accurate

File: sns-book/src/domain-name/domain-find-for-owner.md [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+
+```js
+domains.map((domain) => domain.toBase58());
+```
+
+Another option to retrieve not only human readable Public Keys, but also the domain name in a human readable string format is to use the following function from spl-name-service library below.
```
Comment (dr497) [inline sns-book/src/domain-name/domain-find-for-owner.md]: 1. Same as above for `human readable Public Keys` it reads a bit weird

File: sns-book/src/domain-name/domain-find-for-owner.md [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+
+```js
+const domainKeysWithStrings = getDomainKeysWithReverses(connection, user);
```
Comment (dr497) [inline sns-book/src/domain-name/domain-find-for-owner.md]: Variable name `domainKeysWithStrings` is a bit misleading

File: sns-book/src/domain-name/domain-find-for-owner.md (hunk 1 above)
Comment (akashrdev) [reply sns-book/src/domain-name/domain-find-for-owner.md]: 1.) This is a typo, I'll update
2.) I'll update this as well, your description describes the method better

File: sns-book/src/domain-name/domain-find-for-owner.md (hunk 2 above)
Comment (akashrdev) [reply sns-book/src/domain-name/domain-find-for-owner.md]: Maybe something like, "Another option to retrieve public keys, as well as  their corresponding domain names in a string format is to use..."

File: sns-book/src/domain-name/domain-find-for-owner.md (hunk 3 above)
Comment (akashrdev) [reply sns-book/src/domain-name/domain-find-for-owner.md]: As you mentioned above, reverses is known to be the human readable format so I think domainsWithReverses will be a better variable name.

**Review by dr497 — APPROVED** [review]

**Pull request merged**